test/puppet-mode-test.el

```diff
@@ -601,6 +601,82 @@ package { 'bar':
     (should (local-variable-p 'imenu-create-index-function))
     (should (equal imenu-create-index-function #'puppet-imenu-create-index))))
 
+
+;;;; Indentation
+
+(defmacro flycheck-ert-with-temp-buffer (&rest body)
```

Comment:
Yes, copypasta.

I've never user ert (or much of elisp) before and with little educated googling those seemed useful for the kinds of testing setup I thought was appropriate here.